static/app/components/events/interfaces/spans/spanTreeModel.spec.tsx

```diff
@@ -550,6 +550,10 @@ describe('SpanTreeModel', () => {
       },
     };
 
+    if (!Array.isArray(event2.entries[0].data)) {
```

Comment:
request transaction entry is not an array so we need to guard from push on non array value (doing so narrows the type to spans[])